core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -259,99 +250,113 @@ oppia.controller('StateResponses', [
+          'AngularNameService', 'DEFAULT_OUTCOME_CLASSIFICATION',
           function(
-              $scope, $injector, $uibModalInstance,
-              ExplorationHtmlFormatterService,
+              $scope, $injector, $uibModalInstance, $http,
+              ExplorationHtmlFormatterService, TrainingModalService,
               stateInteractionIdService, stateCustomizationArgsService,
               ExplorationContextService, EditorStateService,
               ExplorationStatesService, TrainingDataService,
-              AnswerClassificationService, FocusManagerService,
-              AngularNameService, EXPLICIT_CLASSIFICATION) {
+              AnswerClassificationService, EXPLICIT_CLASSIFICATION,
+              UrlInterpolationService, TRAINING_DATA_CLASSIFICATION,
+              AngularNameService, DEFAULT_OUTCOME_CLASSIFICATION) {
             var _explorationId = ExplorationContextService.getExplorationId();
             var _stateName = EditorStateService.getActiveStateName();
             var _state = ExplorationStatesService.getState(_stateName);
-
-            $scope.stateContent = _state.content.getHtml();
-            $scope.inputTemplate = (
-              ExplorationHtmlFormatterService.getInteractionHtml(
-                stateInteractionIdService.savedMemento,
-                stateCustomizationArgsService.savedMemento,
-                false,
-                'testInteractionInput'));
-            $scope.answerTemplate = '';
-
-            $scope.trainingData = [];
-            $scope.trainingDataAnswer = '';
-            $scope.trainingDataFeedback = '';
-            $scope.trainingDataOutcomeDest = '';
-
-            // Retrieve the interaction ID.
             var interactionId = stateInteractionIdService.savedMemento;
 
-            var rulesServiceName =
+            var rulesServiceName = (
               AngularNameService.getNameOfInteractionRulesService(
-                interactionId);
+                interactionId));
 
             // Inject RulesService dynamically.
             var rulesService = $injector.get(rulesServiceName);
 
-            // See the training panel directive in StateEditor for an
-            // explanation on the structure of this object.
-            $scope.classification = {
-              answerGroupIndex: 0,
-              newOutcome: null
+            var fetchAndShowUnresolvedAnswers = function(expId, stateName) {
+              var unresolvedAnswersUrl = UrlInterpolationService.interpolateUrl(
+                '/createhandler/get_top_unresolved_answers/<exploration_id>', {
+                  exploration_id: expId
+                });
+              $http.get(unresolvedAnswersUrl, {
+                params: {
+                  state_name: _stateName
+                }
+              }).success(function(response) {
+                $scope.showUnresolvedAnswers(response.unresolved_answers);
+              }).error(function(response) {
+                $scope.showUnresolvedAnswers([]);
```

Comment:
Shouldn't some kind of error be displayed here (or at least a $log.error() to record this in the backend)?

Reply:
Done. Added $log.error()